src/main/java/graphql/annotations/processor/GraphQLAnnotations.java

```diff
@@ -73,6 +76,13 @@ public GraphQLAnnotations() {
         extensionsHandler.setMethodSearchAlgorithm(methodSearchAlgorithm);
         extensionsHandler.setFieldRetriever(fieldRetriever);
 
+        GraphQLAdditionalTypesHandler additionalTypesHandler = new GraphQLAdditionalTypesHandler();
+        additionalTypesHandler.setGraphQLInterfaceRetriever(interfaceRetriever);
+        additionalTypesHandler.setGraphQLObjectInfoRetriever(objectInfoRetriever);
+        additionalTypesHandler.setMethodSearchAlgorithm(methodSearchAlgorithm);
+        additionalTypesHandler.setFieldSearchAlgorithm(fieldSearchAlgorithm);
+
```

Comment:
Can you also add the GraphQLAdditionalTypesHandler to GraphQLAnnotationsComponent ? A simple field with getter/setter/unsetter as the others fields should be ok

Reply:
Can you tell me exactly what to do? I have no idea in OSGI

Reply:
I can do a PR once it will be merged